changelog.md

```
@@ -1,5 +1,11 @@
 # Changelog
 
+### [5.29.3] 2026-08-18
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)